03/app.js

```diff
@@ -2,30 +2,65 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 class Article extends React.Component {
-    state = {
-        comments: [],
+    constructor(props) {
+        super(props)
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.createList = this.createList.bind(this);
+        this.state = {
+            comments: ["komentarz"],
+            textValue: ""
+        }
     }
-    
+    handleSubmit(event) {
+
+        event.preventDefault();
+
+        if (this.state.textValue !== "") {
+            this.state.comments.push(this.state.textValue);
```

Comment:
Bardzo poważny błąd!
Tylko w `constuctor` używamy `this.state = /* ... */`, aby ustawić nowe wartości dla `state`.
Poza musimy użyć `this.setState()` czyli `this.setState(state => [...this.state.comments, this.state.textValue])`;
Powyższy kod oznacza:
- `...this.state.commnets` - tworzymy kopię poprzedniego state
- `[...this.state.comments, this.state.textValue]` - uzupełniany kopię o nowy element